application/backup.sh

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env sh
```

Comment:
Because `local` is used, you should explicitly use bash shell.
```suggestion
#!/usr/bin/env bash
```